questdb-rs/src/ingress/mod.rs

```diff
@@ -251,12 +251,11 @@ impl From<Infallible> for Error {
     }
 }
 
-fn write_escaped_impl<Q, C>(check_escape_fn: C, quoting_fn: Q, output: &mut String, s: &str)
+fn write_escaped_impl<Q, C>(check_escape_fn: C, quoting_fn: Q, output_vec: &mut Vec<u8>, s: &str)
```

Comment:
For consistency with the other functions, you might want to rename `output_vec` to `output`.